data_labeling_examples/bulk_labeling_python/config.py

```diff
@@ -0,0 +1,30 @@
+import multiprocessing as mp
+# for help, run:
+# python3 help.py
+
+# config file path
+CONFIG_FILE_PATH = "~/.oci/config"
+# config file profile
+CONFIG_PROFILE = "DEFAULT"
+# region identifier of DLS Dataset
+REGION_IDENTIFIER = "uk-london-1"
+# service_endpoint
+SERVICE_ENDPOINT_DP = f"https://dlstest-dp.{REGION_IDENTIFIER}.oci.oraclecloud.com"
+# compartment where DLS Dataset exists
+COMPARTMENT_ID = "ocid1.compartment.oc1..aaaaaaaamrnszmfhl7xzawtef3rl6gmsb62tcoag32yiv5ujorognkuzrqfq"
+# ocid of the DLS Dataset
+DATASET_ID = "ocid1.datalabelingdatasetint.oc1.uk-london-1.amaaaaaaniob46iarr2zttq7c5th3jfqwab7d3vrq4daa52tcnnwhkgrowca"
+# an array where the elements are all of the labels that you will use to annotate records in your DLS Dataset with.
+# Each element is a separate label.
+LABELS = ["dog", "cat"]
+# the algorithm that will be used to assign labels to DLS Dataset records
+LABELING_ALGORITHM = "FIRST_LETTER_MATCH"
```

Comment:
Please mention possible values for labeling algorithms in this comment.

Reply:
Comment -> [" ENUMS for labeling algorithm "FIRST_LETTER_MATCH", "FIRST_REGEX_MATCH", "CUSTOM_LABELS_MATCH" ]

Reply:
I meant that you can provide the possible values as comment in the config's comment for easier resference by user.